pkgs/applications/graphics/ImageMagick/default.nix

```diff
@@ -77,7 +77,8 @@ stdenv.mkDerivation rec {
   '';
 
   meta = with lib; {
-    homepage = "http://www.imagemagick.org/";
+    homepage = "https://legacy.imagemagick.org/";
+    changelog = "https://legacy.imagemagick.org/script/changelog.php";
```

Comment:
```suggestion
    homepage = "https://imagemagick.org/";
    changelog = "https://imagemagick.org/script/changelog.php";
```

Reply:
https://imagemagick.org/ is for version 7, https://legacy.imagemagick.org/ for version 6